src/services/infra/InfraService.ts

```diff
@@ -305,7 +274,57 @@ export default class InfraService {
     }
   }
 
+  // special case for MOE folks, not to send email to the agency directly.
+  isMOEEmail = (email: string) => {
+    const moeEmail = "moe.edu.sg"
+    return email.substring(email.length - moeEmail.length) === moeEmail
+  }
+
   pollQueue = async () => {
     setInterval(this.siteLaunchUpdate, SITE_LAUNCH_UPDATE_INTERVAL)
   }
+
+  private async sendStatusEmail(message: MessageBody, site: Site) {
+    const emailDetails: { subject: string; body: string } = {
+      subject: "",
+      body: "",
+    }
+    let params
```

Comment:
this causes `params` to be typed as `any` - we should instead also use a ternary here to assign

Reply:
7bcf556